src/test/scala/freeeventsourcing/eventselector/AggregateTypeEventSelectorTests.scala

```diff
@@ -0,0 +1,91 @@
+package freeeventsourcing.eventselector
+
+import freeeventsourcing._
+import freeeventsourcing.EventSelector.WithEventType
+import freeeventsourcing.EventSelector.ops._
+import freeeventsourcing.accountprocessing.Account.Event.{ Closed, Opened }
+import freeeventsourcing.accountprocessing.Transaction.Event.Created
+import freeeventsourcing.accountprocessing.{ Account, Transaction }
+import freeeventsourcing.utils.CompositeName
+import org.scalatest.{ FlatSpec, Matchers }
+
+class AggregateTypeEventSelectorTests extends FlatSpec with Matchers {
+  val router = AggregateTypeEventSelector.Router.forAggregateType(Account)
+
+  val openedEvent = AggregateEvent[Account.type, Opened](Account, Account.Id(1), Opened("Mario"))
+
+  "AggregateTypeEventSelector.Router " should " produce the same topic as the matching AggregateTypeEventSelector" in {
+    val toIndex = router.apply(openedEvent)
+    toIndex.size shouldBe 1
+    val indexed = toIndex.head
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of head on collections.](https://www.codacy.com/app/msiegenthaler/free-event-sourcing/file/2654090849/issues/source?bid=3298199&fileBranchId=3311368#l20)